flask_s3.py

```diff
@@ -39,6 +41,48 @@
 
 __version__ = (0, 2, 10)
 
+FLASKS3_PREFIX_DEFAULT = ''
+FLASKS3_VERSION_FUNCTION_DEFAULT = None
+
+
+def _get_versioning_prefix(app):
```

Comment:
Does this really need to be a function? Especially since you call it later with zero arguments, a string would probably suffice.